atlante/filestore/s3/s3.go

```diff
@@ -73,7 +88,10 @@ func initFunc(cfg filestore.Config) (filestore.Provider, error) {
 		intermediate:         intermediate,
 		intermediateBucket:   intermediateBucket,
 		intermediateBasePath: intermediateBasePath,
-		uploader:             uploader,
+		s3:                   s3.New(sess),
+		uploader:             s3manager.NewUploader(sess),
+		urlTimeout:           time.Duration(urlTimeout) * time.Minute,
+		genPresignedURL:      genPresigned,
```

Comment:
I don't think we need this flag. I think we can just use if s3 is nil or not. The flag should be used here to determine weather we populate the s3 object or not.